Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -279,6 +265,11 @@ - (void)setupToolbar {
 }
 
 - (void)updateToolbarItemsIfNeeded {
+    
+    if (!self.saveButtonController) {
```

Comment:
i see what you mean. how about just creating it in `init` w/ the `articleTitle` and setting the save button on it in `viewDidLoad`? best of both worlds?